api/models/server.js

```diff
@@ -112,7 +112,6 @@ exports.deleteScoutAdminUser = function(url, username, password){
   if (!process.env.SCOUT_ADMIN_USER_NAME) {
     scoutAdminUserName = 'ScoutAdmin';
   }
-  console.og
```

Comment:
This line was seriously in master and it still built? wouldn't have expected that

Reply:
Yeah haha. And that was the only issue. I have no idea how that worked...